Cleanup

wesley-tang opened the pull request:
Mostly minor changes in terms of whitespace and the use of `let`s and `const`s.

### Endpoint Change

The **/all/article route** now returns _ONLY A STRING_ containing the `text` of the article.
**NOTE**: If the article fails to scrape, an empty string will be returned.

wesley-tang committed:
Cleaned variables and updated article endpoint

wesley-tang committed:
Convert to config for local Port.

Converted to using config file for setting the local port. Also minor changes in fixing bug and adding consts instead of lets.

wesley-tang committed:
Fix whitespace and minor style change

Added spaces before opening curly braces and changed check if port exists to typical javascript style

brian-norman commented on lib/prepArticles.js:
```diff
@@ -49,7 +49,7 @@ function prepAll(store, callback) {
 		            }
 
-		            if (completedArticles === articleList.length){
+		            if (completedArticles >= articleList.length){
```
Curious what this is for?

wesley-tang replied:
Oh lol bad logic on my part, it's to account for Cole's encounter with the completed articles somehow exceeding the total amount, just in case.

brian-norman reviewed (approved):
Awesome very clean, test and release 🚀 Rebase first!

wesley-tang committed:
Resolve conflicts for cleanup branch merge

Resolved conflicts that occured when merging the cleanup branch to master

wesley-tang committed:
Resolve conflicts for cleanup branch merge

Resolved conflicts that occured when merging the cleanup branch to master. Oops missed a file. LOL

wesley-tang committed:
Update console logs to reflect multiple categories

Since each category happens async, the console logs were confusing, so the category of each piepline is noted now.
